m4_sessions/1901-inning/social-movements.md

```diff
@@ -11,82 +11,57 @@ In an increasingly technological world, there is an opportunity for those of us
-  - Example project that David wants to start working on after Turing
-  - Quick go-around: what projects jumped out to you when looking through _Code For Social Good_
-
-**3. Activate:** (25 minutes)
+**2. Inform:** (15 minutes)
 
-Individual Exercise within Groups 
+_Present for 5 mins (Meghan, Maria)_
+Talking about what social movements are and why they're important, and how we can power the movements.
+Examples of social movements and the role of technology 
 
-_We'll split the room into 5 groups and each of the presenters will sit with a group to push the conversation forward._
+_Present for about 1 min each on what they are, why they're good/bad, and how they've used tech (about 10 mins total)(Isaac)_
 
-- (10 minutes) Think about a movement you are passionate about and think about ways you can be involved as a developer - the distinction being that we are focused on how we can use our skills to further the development of technology that helps social movements for good (not use technology)
-  - ****Worksheet****
-    - Who/what are the sides? What are their perpsectives/motivations?
-    - What is being tried/has already been tried? 
-    - How can technology help push the movement forward?
+Good:
+BLM- (https://en.wikipedia.org/wiki/Black_Lives_Matter)
+Environmental movements- (https://en.wikipedia.org/wiki/Environmental_movement)
+LGBT movements- (https://en.wikipedia.org/wiki/LGBT_social_movements)
+Me too- (https://en.wikipedia.org/wiki/Me_Too_movement)
 
-- (5 minutes) Write down some committments you want to make to being an active voice in/contributor to the movement/s you care about
+Bad:
+9/11 Theories- (https://en.wikipedia.org/wiki/9/11_Truth_movement)
+Info wars- Banned on youtube (https://en.wikipedia.org/wiki/InfoWarsZ)
+(Look for more bad examples)
 
-- (10 minutes) Small group share out & discussions (3-4 people max)
+(Pom 5 mins)
 
+**3. Activate:** (40 minutes)(Ashley)
+If 14 people:
+4 groups of 3 and one group of 2.
+If 15 People:
+3 groups of 4 and 1 of 3
+If 16 people:
+4 groups of 4
+Individual Exercise within Groups 
 
-**4. Share out:** (10 minutes)
-- What is your personal committment (post Turing) to benefit the greater good as a developer (i.e. with your new skills)? 
 
+(10 minutes) 
+Think about a movement or a topic you as a group are passionate about that has to do with social change and think about the ways you can be involved as a developer and write down some notes/thoughts.
 
+(20 minutes) 
+Pick a social movement that your group is passoniate about (either a movement from the list or a movement you came up with), and come up with an app idea that contributes to this movement/topic in a positive way. Create a simple wireframe and a storyboard.
 
+(10 minutes) 
+Pitch your app idea to the group, including what social movement this is geared to impower, the purpose of the app. Appoint one person to speak about the app.
```

Comment:
I like this...you can also have them join your channel and then share ideas/apps in the channel for continued conversation.  Maybe close with a way for them to identify next steps.